examples/loss_and_gradient_se3.py

```diff
@@ -110,7 +111,7 @@ def main():
     scalar = gs.array(cos)
     vec = sin * u
     translation = gs.array([5., 6., 7.])
-    y_pred_quaternion = gs.hstack([scalar, vec, translation])
+    y_pred_quaternion = gs.concatenate([[scalar], vec, translation], axis=0)
```

Comment:
why the `[]` around scalar?

Reply:
Scalar tensors does not have a shape (returns ```shape=()``` or ```TensorShape([])```), so it's not possible to concatenate with tensor vectors. The square bracket assigns it with ```shape=(1,)``` or ```TensorShape([Dimension(1)])```